server/utils/redditcrawler.js

```diff
@@ -0,0 +1,42 @@
+var snoowrap = require('snoowrap');
+const { createMention, getMention } = require("../models/mention-model");
+const r = new snoowrap({
+  userAgent: 'webcrawler',
+  clientId: 'rwl4j4FrYnxqPA',
+  clientSecret: 'qZ1p1Cp8q2Va6gBv8A18oI2KZGrK0Q',
+  username: 'bot3424',
+  password: 'bot3424'
```

Comment:
Remember to move this info to an environment variable (inside the `.env` file). Otherwise, as it stands, anyone that sees this code on GitHub can hijack your Reddit developer info and use it for whatever they want.